### PR TITLE
Graceful messages when db credentials are wrong

### PR DESCRIPTION
**Pull request by vitsalis**

### CODE REVIEW

**Comment by vitsalis** [comment]
Fixes #62

File: models/database.php [hunk 1]
```diff
@@ -1,9 +1,12 @@
 <?php
+    function throwDBException( $message ) {
+        throw new DBException( $message );
+    }
     function dbInit() {
         global $config;
 
-        mysql_connect( $config[ 'db' ][ 'host' ], $config[ 'db' ][ 'user' ], $config[ 'db' ][ 'pass' ] ) or die( 'Failed to connect to MySQL: ' . mysql_error() );
+        mysql_connect( $config[ 'db' ][ 'host' ], $config[ 'db' ][ 'user' ], $config[ 'db' ][ 'pass' ] ) or throwDBException( 'Failed to connect to MySQL: ' . mysql_error() );
```
Comment (pkakelas) [inline models/database.php:8]: I believe we should have a different exception for situations like wrong credentials e.t.c.

Comment (vitsalis) [inline models/database.php:8]: `mysql_error()` covers that I think

File: models/database.php [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
 
-        mysql_select_db( $config[ 'db' ][ 'dbname' ] ) or die( 'Failed to select MySQL database: ' . mysql_error() );
+        mysql_select_db( $config[ 'db' ][ 'dbname' ] ) or throwDBException( 'Failed to select MySQL database: ' . mysql_error() );
```
Comment (dionyziz) [inline models/database.php:10]: Why don't you just `throw` and need a function?

**Pull request merged**